third_party/mlir/BUILD

```diff
@@ -38,6 +38,65 @@ filegroup(
     ],
 )
 
+PYBIND11_COPTS = select({
+    "@tf_runtime//:windows": [
```

Comment:
Are those the same as the other two? If so, can we share them?

Presumably they would best fit in the tensorflow/toolchains repo, but tfrt does not currently depend on it. Maybe just have it in this repo somewhere with a comment where it should go eventually?

Reply:
Yes, this is one place where we could use TFRT_OPTS. I was thinking to keep that factoring out in a follow up PR, since there may be other changes (including potential differences in flags).

Reply:
Sounds good.